### PR TITLE
[ARTEMIS] Benchmark in Artemis Test

### PR DESCRIPTION
**Pull request by AzHicham**
Benchmark in Artemis Test using pytest-benchmark
We measure time to compute a request !

### CODE REVIEW

**Comment by AzHicham** [comment]
image

Sample report

File: artemis/base_pytest.py [hunk 1]
```diff
@@ -664,3 +649,33 @@ def print_diff(ref_file, resp_file, test_name):
             # print_diff(response_filepath, output_reference_filepath, self.get_test_name())
 
             raise
+
+    def request_compare(self, http_query, checker):
```
Comment (xlqian) [inline artemis/base_pytest.py:653]: is this function still used anywhere else??

Comment (AzHicham) [reply artemis/base_pytest.py:653]: request_compare is still called by _api function, but my next commit will remove it, allowing us to benchmark api call too.

**Review by pbench — APPROVED** [review]
:+1:

**Review by xlqian — APPROVED** [review]

File: Jenkinsfile [hunk 2]
```diff
@@ -5,7 +5,7 @@ pipeline {
         string(name: 'artemis_branch', defaultValue: 'master', description: 'Artemis branch to checkout')
         string(name: 'artemis_data_repo', defaultValue: 'CanalTP/artemis_data', description: 'Artemis_data github repository ')
         string(name: 'artemis_data_branch', defaultValue: 'master', description: 'Artemis_data branch to checkout')
-        string(name: 'artemis_ref_repo', defaultValue: 'pbench/artemis_references', description: 'Artemis_references github repository ')
+        string(name: 'artemis_ref_repo', defaultValue: 'AzHicham/artemis_references', description: 'Artemis_references github repository ')
```
Comment (xlqian) [inline Jenkinsfile:8]: ?

Comment (AzHicham) [reply Jenkinsfile:8]: I forked CanalTP/artemis_references and created a branch with new ref for Artemis NG (Artemis NG Json results are different from Artemis Old, floating point precision ....)

**Pull request merged**